### ISSUE TITLE
Adding assets to open project

### ISSUE THREAD

**Issue by apoisonedgift** [master-technology/AssetManagerStudio#14]
I noticed that if you try to add an asset via AMS to a project that is open, it simply won't work. The epic launcher allows you to do this and while I can understand why it might be disabled - there's nothing to notify you that is why it won't add. If you want to ensure people don't add assets to open projects I can understand and get behind that - but it'd be useful to have some feedback just saying "project is open, close the project and try again" or something because it took me a while to work out what the issue was haha

**Comment by NathanaelA** [comment]
Strange, I don't have anything specifically blocking adding to a project that is open.   I'll run some tests and see if I can figure out what isn't working...

**Comment by apoisonedgift** [comment]
I guess its possible that something else is stopping it but every time I try to add an asset it just does nothing when its open and it seems to then work fine if I close the project haha

**Comment by NathanaelA** [comment]
@apoisonedgift - I just tested this on my Linux machine using Unreal 5.2 (with the Cropout sample project).  I added a different asset and it showed up immediately in the content browser while the cropout sample was open.   
- Which version of Unreal
- Windows or Linux
- Anything else you can think of that might be unique?

**Comment by apoisonedgift** [comment]
Sorry for taking a few days to get back to you here... So it was happening in 5.0 and 5.2 (hadn't tried a project in 5.1) with multiple projects, on windows. Initially I noticed it with my own marketplace assets in trying to make tutorial content for them - but I tested with a number of other assets and saw the same behaviour. I can't really think of anything else that is unique as I tried multiple projects and multiple assets.

That said, I went to do some specific testing today in order to see if I could narrow it down to anything else and saw that you had released an update to 0.0.17 and since performing the update, I cannot recreate the behaviour so... idk if you changed something or it was a random bug that got fixed by upgrading or what but... at the moment, at least, I cannot recreate this issue. Success? haha
